### PR TITLE
[WIP] feat: RWA Editor integration with RWA document

### PR DESCRIPTION
**Pull request by gpuente**
## Description:

- Added `RWA Editor` component

depends on https://github.com/powerhouse-inc/design-system/pull/186

need to do a release of `design-system` and update the dep in the package.json here

### CODE REVIEW

File: editors/rwa/portfolio.tsx [hunk 1]
```diff
@@ -1,67 +1,126 @@
...
-        reset,
-    ) => {
-        // TODO: dispatch create asset action
-        console.log('onSubmitCreate', data);
-        console.log('reset', reset);
-        setShowNewAssetForm(false);
-    };
+    const toggleExpandedRow = useCallback(
+        (id: string) => {
+            setExpandedRowId(id === expandedRowId ? undefined : id);
```
Comment (acaldas) [inline editors/rwa/portfolio.tsx:71]: Small nitpick but here I would use an updater function instead.
`setExpanderRowId(expandedRowId => id === expandedRowId ? undefined : id);`
Then the dependency on `expandedRowId` can be removed from the hook.

File: editors/rwa/portfolio.tsx [hunk 2]
```diff
@@ -1,67 +1,126 @@
...
+        },
+        [expandedRowId],
+    );
+
+    const onClickDetails: FixedIncomeAssetsTableProps['onClickDetails'] =
+        useCallback(
+            item => {
+                setExpandedRowId(
```
Comment (acaldas) [inline editors/rwa/portfolio.tsx:79]: Same as above

File: editors/rwa/portfolio.tsx (hunk 1 above)
Comment (ryanwolhuter) [reply editors/rwa/portfolio.tsx:71]: all good, will update

File: editors/rwa/portfolio.tsx (hunk 2 above)
Comment (ryanwolhuter) [reply editors/rwa/portfolio.tsx]: this one was actually a residual that does redundant logic. right now the only thing that needs to happen when the user clicks on the details arrow is to expand / shrink the accordion. I've removed the redundant logic and left the callback here for if someone needs to do more by accessing the item.

**Pull request merged**